src/Classes/Particle.ts

```diff
@@ -175,11 +177,6 @@ export class Particle implements IParticle {
 			y: 0,
 		};
 
-		/* check position - avoid overlap */
```

Comment:
This overlap check will replace the position with a free one. Does this code is moved somewhere else?

Reply:
Ah, I misunderstood the function to be that of the collision detection, no worries I can roll that back. Preferably, I might find an alternative solution since I feel its the job of the creator of the particle to make such necessary checks.